services/web/server/src/simcore_service_webserver/api/v0/openapi.yaml

```diff
@@ -9549,13 +9549,9 @@ components:
       properties:
         name:
           title: Name
-          maxLength: 100
-          minLength: 1
           type: string
         description:
           title: Description
-          maxLength: 100
-          minLength: 1
```

Comment:
why is this not updated?

Reply:
Because it is a truncation. There is **no constraint** or limit. If it appears in the OAS, then the system should return an error! 
There is a trick to truncate in the pre-processor before it gets validated but I think it would be confusing